docs/en-US/component/use-upload.md

```
@@ -77,4 +77,84 @@ abort()
 | compressed | Whether to compress video | boolean | true | - |
 | maxDuration | Maximum video duration (seconds) | number | 60 | - |
 | camera | Camera direction | 'back' \| 'front' | 'back' | - |
-| extension | Filter by file extension (H5 supports all type filtering, WeChat Mini Program supports filtering when all and file, other platforms do not support) | string[] | - |
\ No newline at end of file
+| extension | Filter by file extension (H5 supports all types, WeChat Mini Program supports filtering when all and file, other platforms do not support) | string[] | - |
```

markdownlint-cli2 (0.17.2)

Table column count
Expected: 5; Actual: 4; Too few cells, row will be missing data

(MD056, table-column-count)